core/src/main/resources/jenkins/security/csp/impl/CspRecommendation/index.properties

```diff
@@ -0,0 +1,6 @@
+# TODO Add better explanation
+paragraph1 = Content Security Policy is a security mechanism that can reduce or eliminate the impact of web security vulnerabilities like cross-site-scripting (XSS).
+paragraph2 = Most popular Jenkins plugins are compatible with Jenkins's default rule set, but not everything currently installed may be. \
+  If you choose to enforce CSP and it causes the Jenkins UI to break, you can start Jenkins with the Java system property \
```

Comment:
This should be linked to the page https://www.jenkins.io/doc/book/managing/system-properties/ so that the person knows how to configure a system property.

Reply:
Thanks. Please note that the content of this page is still placeholder content. I'm currently debating either making it much more extensive, or alternatively linking to the docs on jenkins.io. In the latter case, this might end up becoming just a popup dialog instead.